app/views/layouts/application.html.erb

```diff
@@ -26,14 +26,25 @@
             <%= link_to "#{icon('collection', 'Collections')} Collections".html_safe, collections_path, class: (current_page?(collections_path) ? "nav-link active" : "nav-link") %>
           </li>
         <% end %>
+        <li class="nav-item">
+          <%= link_to "#{icon('box-seam', 'Models')} Models".html_safe, {controller: 'models'}, class: (current_page?(models_path) ? "nav-link active" : "nav-link") %>
+        </li>
         <% Library.all.each do |library| %>
           <li class="nav-item">
-            <%= link_to "#{icon('box-seam', '#{library.name}')} #{library.name}".html_safe, library, class: (current_page?(library) ? "nav-link active" : "nav-link") %>
+            <%= link_to "#{icon('box-seam', '#{library.name}')} #{library.name}".html_safe, {controller: 'models', library: library.id}, class: ((current_page?(library) || (library.id.to_s == params[:library])) ? "nav-link active" : "nav-link") %>
           </li>
         <% end %>
         <li class="nav-item me-auto">
           <%= link_to icon("plus-circle", "New Library"), new_library_path, class: (current_page?(new_library_path) ? "nav-link active" : "nav-link") %>
         </li>
+        <li class="nav-item">
+          <%= form_with url: models_path, method: :get, class: "mt-0" do |f| %>
```

Comment:
The styling on this search box is a bit odd in my browser - though you don't need to address that, I'm happy to do a styling pass after merging.

Reply:
Yeah... I figured it would need a styling pass after the fact.    First time I've touched bootstrap either :)